admin/conference-template/params.data

```diff
@@ -0,0 +1,82 @@
+{
+    /*
+    CONFERENCE ID
+
+    The OpenReview group ID of this conference. This should exactly match the
+    directory path that this file is in.
+
+    E.g. ICLR.cc/2018/Conference
+    */
+    "<<CONFERENCE_ID>>": "TestConf.org",
+
+
+    /*
+    A short phrase that refers to the conference, to be used in email
+    correspondence and other times when a shorter version of the full
+    conference name is needed.
+
+    E.g. for the 2018 International Conference on Learning Representations,
+    a SHORT_PHRASE could be "ICLR 2018"
+    */
+    "<<SHORT_PHRASE>>": "TestConf 2018",
+
+
+    /*
+    SUBMISSION NAME
+
+    The "name" of the submission invitation. Note that this is different from
+    the submission invitation ID. The name is used to create the submission
+    invitation ID with the following pattern:
+
+    submission invitation ID = <<CONFERENCE_ID>>/-/<<SUBMISSION_NAME>>
+
+    E.g. ICLR.cc/2018/Conference/-/Submission
+    */
+    "<<SUBMISSION_NAME>>": "Submission",
+
+
+    /*
+    SUBMISSION TIMESTAMP
+
+    The timestamp of the submission duedate, in unix epoch time.
+    Should be entered here as a 13-digit integer representing the number of
+    milliseconds since 00:00:00 Coordinated Universal Time (UTC).
+
+    REVIEW_TIMESTAMP
+    Same as above, but for the review duedate.
+    */
+    "<<SUBMISSION_TIMESTAMP>>": "1607225802000",
+    "<<REVIEW_TIMESTAMP>>": "1607225802000",
+
+
+    /*
+    Conference Homepage Fields
+
+    TITLE: The full, main title of the conference. Appears in bold at the top
+    of the conference homepage. e.g. "ICLR 2018 Conference Track"
+
+    SUBTITLE: The full subtitle of the conference. Appears as a subtitle to
+    the TITLE.
+    e.g. "The 6th International Conference on Learning Representations"
+
+    LOCATION: A string describing the location of the conference.
+    e.g. "Vancouver Convention Center, Vancouver, BC, Canada"
+
+    EVENT_DATE: A string describing the date of the actual conference event.
+    e.g. "April 30 - May 3, 2018"
+
+    URL: A url for the conference website.
+    e.g. "http://www.iclr.cc"
+
+    DEADLINE: A string describing the deadline of the conference.
+    e.g. "Submission Deadline: 5:00PM Eastern Standard Time, October 27, 2017"
+
+    */
+
+    "<<TITLE>>": "TITLE",
```

Comment:
I would remove '<<' and '>>' from the keys and add them when we build the templates with the configuration. It is more descriptive to have the name only.